aws_longer/__init__.py

```diff
@@ -107,14 +107,18 @@ def handle_cleanup(arguments):
     except keyring.errors.PasswordDeleteError:
         pass
 
+def get_aws_username() -> str:
+    '''Retrieve canonical aws username for session identifiers.'''
+    username = boto3.client('iam').get_user()['User']['UserName']
+    return username
```

Comment:
Can you use the existing session to create the client?

```suggestion
    return boto3_session().client("iam").get_user()["User"]["UserName"]
```

Reply:
sure